js/popup.js

```diff
@@ -6,18 +6,37 @@ window.addEventListener('load', () => {
   let pageUrl = document.getElementById("page-url");
   let pageComment = document.getElementById("page-comment");
 
+  const findData = (items, url) => {
+    return items.data.filter((item) => item.url === url)[0] || null
+  }
+
   chrome.tabs.getSelected(null, (tab) => {
     pageTitle.value = tab.title;
     pageUrl.textContent = tab.url;
+    new Promise((resolve, reject) => {
+      chrome.storage.sync.get('data', resolve)
+    }).then((items) => new Promise((resolve, reject) => {
```

Comment:
[nits] 好みの問題なのでおまかせしますー
```suggestion
    }).then(items => new Promise((resolve, reject) => {
```